account_cutoff_prepaid/account_cutoff_view.xml

```diff
@@ -54,6 +55,7 @@
     <field name="arch" type="xml">
         <field name="parent_id" position="after">
             <field name="move_line_id" invisible="'prepaid' not in context.get('type', '-')"/>
+            <field name="move_line_period_id" invisible="'prepaid' not in context.get('type', '-')"/>
```

Comment:
Just a little remark here...
With this new field the treeview now contains 4 unstorable fields ! 
No fears about performance ?
No needs to sort the tree on these fields ?